libraries/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -235,7 +235,9 @@
     #define EMPTY_ELSE_MARKER
 #endif
 
-#define IOT_MQTT_SERVER_MAX_CLIENTID_LENGTH                    ( ( uint16_t ) 23 ) /**< @brief Optional maximum length of client identifier specified by MQTT 3.1.1. */
+#define IOT_MQTT_SERVER_MAX_CLIENTID_LENGTH                    ( ( uint16_t ) 23 )          /**< @brief Optional maximum length of client identifier specified by MQTT 3.1.1. */
```

Comment:
I think we can shorten these macro names.

So just `MQTT_MAX_CLIENTID_LENGTH`, `MQTT_MAX_PUBLISH_LENGTH`, etc.

Reply:
Changed